getyour/app/backend.py

```diff
@@ -551,7 +551,7 @@ def map_iq_enrollment_status(program):
     try:
         if program.is_enrolled and program.has_renewed:
             return "ACTIVE"
-        elif not program.is_enrolled and not program.has_renewed:
+        elif not program.is_enrolled:
```

Comment:
Because we're setting the `has_renewed` field to `true` when a user initially applies for an IQ program, we no longer need to check it to determine the enrollment status of the program for the user. The enrollment status for a user's program is always 'PENDING' is `is_enrolled` is `false`